srcs/raycasting/dda.c

```diff
@@ -6,93 +6,46 @@
 /*   By: elouisia <elouisia@student.42.fr>          +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2022/10/02 15:40:38 by elouisia          #+#    #+#             */
-/*   Updated: 2022/10/07 13:26:45 by aweaver          ###   ########.fr       */
+/*   Updated: 2022/10/17 16:03:55 by elouisia         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "cub3d.h"
 
-/*
-A_ Cast a ray
```

Comment:
Love this, clear pseudo-code.
Really helpful for me to help review the new code!